examples/go-tc-counter/main.go

```diff
@@ -55,15 +55,37 @@ func main() {
 		action = "sent"
 	}
 
-	var mapPath string
+	var statsMap *ebpf.Map
+
+	opts := &ebpf.LoadPinOptions{
+		ReadOnly:  false,
+		WriteOnly: false,
+		Flags:     0,
+	}
+
+	c := bpfdHelpers.GetClientOrDie()
```

Comment:
I think this only needs to done if we're running on kubernetes (i.e., after the line 69 check below)

Reply:
Actually, the example doesn't work in a non-kube environment when this call is made here.  I got the following error:
```
jordan:~/projects/bpfd/examples/go-tc-counter$ sudo ./go-tc-counter -iface vboxnet1 -direction ingress
2023/03/09 15:10:43 Using Input: Interface=vboxnet1 Priority=50 Source=file:////home/afredette/projects/bpfd/examples/go-tc-counter/bpf_bpfel.o
panic: stat /root/.kube/config: no such file or directory

goroutine 1 [running]:
github.com/redhat-et/bpfd/bpfd-operator/pkg/helpers.getk8sConfigOrDie()
	/home/afredette/projects/bpfd/bpfd-operator/pkg/helpers/helpers.go:65 +0xd9
github.com/redhat-et/bpfd/bpfd-operator/pkg/helpers.GetClientOrDie()
	/home/afredette/projects/bpfd/bpfd-operator/pkg/helpers/helpers.go:79 +0x19
main.main()
	/home/afredette/projects/bpfd/examples/go-tc-counter/main.go:66 +0x270
```

Reply:
And, after I made the suggested changes, the tc example ran fine.  I didn't try the xdp one, but I assume it would be the same.